thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java

```diff
@@ -405,10 +424,10 @@ long createSubscriptionGroup(String yamlAlertConfig) throws ValidationException
   }
 
   @POST
-  @Path("/notification")
+  @Path("/subscription")
```

Comment:
change this path requires frontend changes too.

Reply:
Yes, I have already communicated this with @aaronucsd